### ISSUE TITLE
deepExtend typo

### ISSUE THREAD

**Issue by ionutvmi** [gilbitron/Ideal-Image-Slider#34]
Hi,
I was looking over the code and I noticed a typo `deepExtend` is not defined here

```javascript
	var _deepExtend = function(out) {
		out = out || {};
		for(var i = 1; i < arguments.length; i++){
			var obj = arguments[i];
			if(!obj)
				continue;
			for(var key in obj){
				if(obj.hasOwnProperty(key)){
					if(typeof obj[key] === 'object')
						deepExtend(out[key], obj[key]); // should be _deepExtend
					else
						out[key] = obj[key];
				}
			}
		}
		return out;
	};
```

**Comment by ionutvmi** [comment]
this is still open ?

**Comment by gilbitron** [comment]
Fixed by #69

**Comment by ionutvmi** [comment]
:+1: :smiley:
